dapp/src/pages/DashboardPage/index.tsx

```diff
@@ -1,41 +1,43 @@
 import React from "react"
-import { useMobileMode, useTriggerConnectWalletModal } from "#/hooks"
-import MobileModeBanner from "#/components/MobileModeBanner"
+import { useTriggerConnectWalletModal } from "#/hooks"
 import { featureFlags } from "#/constants"
+import { Grid } from "@chakra-ui/react"
 import DashboardCard from "./DashboardCard"
-import { PageLayout, PageLayoutColumn } from "./PageLayout"
 // import GrantedSeasonPassCard from "./GrantedSeasonPassCard"
 import AcrePointsCard from "./AcrePointsCard"
 import { CurrentSeasonCard } from "./CurrentSeasonCard"
 import BeehiveCard from "./BeehiveCard"
 import UsefulLinks from "./UsefulLinks"
 
-export default function DashboardPage() {
-  const isMobileMode = useMobileMode()
+const TEMPLATE_AREAS_LEDGER_LIVE = `${featureFlags.TVL_ENABLED ? '"dashboard current-season"' : ""}
+                                    ${featureFlags.BEEHIVE_COMPONENT_ENABLED ? '"dashboard beehive"' : ""}
+                                    "dashboard acre-points"
+                                    "dashboard useful-links"`
 
+export default function DashboardPage() {
   useTriggerConnectWalletModal()
 
-  return isMobileMode ? (
-    <MobileModeBanner forceOpen />
-  ) : (
-    <PageLayout>
-      <PageLayoutColumn isMain>
-        <DashboardCard />
-      </PageLayoutColumn>
-
-      <PageLayoutColumn>
-        {featureFlags.TVL_ENABLED && (
-          <CurrentSeasonCard showSeasonStats={false} />
-        )}
-        {/* TODO: Uncomment in post-launch phases */}
-        {/* <GrantedSeasonPassCard /> */}
-        {featureFlags.BEEHIVE_COMPONENT_ENABLED && <BeehiveCard />}
-      </PageLayoutColumn>
-
-      <PageLayoutColumn position="relative">
-        <AcrePointsCard />
-        <UsefulLinks />
-      </PageLayoutColumn>
-    </PageLayout>
+  return (
+    <Grid
+      gridGap={{ base: 4, "2xl": 8 }}
+      gridTemplateColumns={{
+        base: "1fr 1fr",
+        lg: "1fr 31%",
+        "2xl": "1fr 36%",
+      }}
+      templateAreas={TEMPLATE_AREAS_LEDGER_LIVE}
```

Comment:
Same as above. Shouldn't we set this value based on the embed app? If this layout is for all types of apps (standalone and Ledger Live app) we should rename the `TEMPLATE_AREAS_LEDGER_LIVE` to `TEMPLATE_AREAS`.

Reply:
https://github.com/thesis/acre/pull/763/commits/908e8efcec016c8e74ecf84fef59808ad3f651ae